helm-deployment/ing-subchart/templates/ingress-nginx-baremetal.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if eq .Values.ingType "baremetal" -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: ingress-nginx
+  namespace: ingress-nginx
+  labels:
+    app.kubernetes.io/name: ingress-nginx
```

Comment:
looks like annotations used in newer ingress-nginx versions, nginx version we use is 0.17.1
https://github.com/kubernetes/ingress-nginx/blob/nginx-0.17.1/deploy/provider/baremetal/service-nodeport.yaml